Cannot find module 'ziggy-js' or its corresponding type declarations

resources/js/types/global.d.ts(2,37): error TS2307: Cannot find module 'ziggy-js' or its corresponding type declarations. Error: Command "npm run build" exited with 2

--- a/package.json
+++ b/package.json
@@ -22,5 +22,8 @@
         "tailwindcss": "^3.2.1",
         "typescript": "^5.0.2",
         "vite": "^5.0"
+    },
+    "dependencies": {
+        "ziggy-js": "^2.2.1"
     }
 }

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -11,8 +11,7 @@
         "forceConsistentCasingInFileNames": true,
         "noEmit": true,
         "paths": {
-            "@/*": ["./resources/js/*"],
-            "ziggy-js": ["./vendor/tightenco/ziggy"]
+            "@/*": ["./resources/js/*"]
         }
     },
     "include": ["resources/js/**/*.ts", "resources/js/**/*.tsx", "resources/js/**/*.d.ts"]

